tools/config_updater.py

```diff
@@ -560,11 +560,55 @@ def ConfigureEmails(config):
   config.Set("Monitoring.emergency_access_email", emergency_email)
 
 
+def ConfigureUser(config):
+  success = False
+  print
+  username = RetryQuestion("GRR Server Username",
+                           "^[a-z0-9]+$",
+                           config_lib.CONFIG.Get("Server.username"))
+
+  try:
+    pwd.getpwnam(username)
+    print "User %s exists. Continuing..." % username
+    success = True
+  except KeyError:
+    user_create_error = os.system('useradd -r %s -M -d /nonexistant -s '
```

Comment:
/nonexistant is a Debian/Ubuntu convention for users without homedirs.  I'm not terribly opposed to changing it, but I figured I would go with what would look "normal" for the typical system.